packages/framework/resources/views/layouts/publication_list.blade.php

```diff
@@ -0,0 +1 @@
+The default publication list template
```

Comment:
Is this file used?

Reply:
I think this is the only usage (which is not justified as it's in a test) https://github.com/hydephp/develop/blob/b86936c7e8c3ca994cc329285ad5c61419780f25/packages/framework/tests/Feature/Actions/PublicationPageCompilerTest.php#L101

Reply:
Resolved in https://github.com/hydephp/develop/pull/807